sdcm/scan_operation_thread.py

```diff
@@ -490,18 +492,38 @@ def _validate_fullscan_result(
         output = "\n".join([str(i) for i in result])
         if int(result[0].count) <= 0:
             return f"Fullscan failed - count is not bigger than 0: {output}", Severity.ERROR
-        get_query_trace = cmd_result.get_query_trace().events
-        for trace_event in get_query_trace:
-            if dispatch_forward_statement_regex_pattern.search(str(trace_event)):
-                regex_found = True
-                break
         self.log.debug("Fullscan aggregation result: %s", output)
 
-        if not regex_found:
-            self.log.debug("\n".join([str(i) for i in get_query_trace]))
-            self.log.debug("Fullscan aggregation result: %s", output)
-            return "Fullscan failed - 'Dispatching forward_request' message was not found in query trace events", \
-                Severity.WARNING
+        if self.validate_forward_service_requests_counter < 5:
+            self.validate_forward_service_requests_counter += 1
+        else:
+            is_dispatched_forward_request = False
+
+            prometheus = PrometheusDBStats(
+                self.fullscan_params.db_cluster.test_config.tester_obj().monitors.nodes[0].external_address)
+            prometheus_forward_service_requests = prometheus.query(
+                'scylla_forward_service_requests_dispatched_to_other_nodes',
+                self.validate_forward_service_requests_start_time, time.time())
+
+            self.validate_forward_service_requests_counter = 0
```

Comment:
move it right after `else:` so it makes this skip logic more visible

Reply:
I think more visible to keep it  together with resetting the start time:

self.validate_forward_service_requests_start_time = time.time()